### PR TITLE
NTUEEPLUS-5. Add PR workflow documentation

### PR DESCRIPTION
**Pull request by featherchen**
### What is this PR for?

Add PR workflow documentation
### What type of PR is it?
 Documentation

### Todos
* [x] - Pictures

### What is the Jira issue?

https://ntueeplus.atlassian.net/browse/NTUEEPLUS-5

### CODE REVIEW

**Comment by featherchen** [comment]
@noidname01

File: doc/contribution.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+Follow Fork & Pull model
+
+## Step1 Folk(first time)
+
+Folk repo : [EndOdWeb](https://github.com/NTUEE-PLUS/EndOfWeb.git)
```
Comment (noidname01) [inline doc/contribution.md:5]: Typo Bro

**Review by noidname01 — CHANGES_REQUESTED** [review]
Fix Typo

File: doc/contribution.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+Follow Fork & Pull model
+
+## Step1 Fork(first time)
+
+Fork repo : [EndOdWeb](https://github.com/NTUEE-PLUS/EndOfWeb.git)
```
Comment (noidname01) [inline doc/contribution.md:5]: Still Type Od->Of

**Review by noidname01 — CHANGES_REQUESTED** [review]

**Review by noidname01 — APPROVED** [review]
LGTM Thanks for your contribution. @featherchen

**Pull request merged**